.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+        args: ["--target-version", "py35"]
```

Comment:
Target version should be in `pyproject.toml`, and ideally not also in other places.

Reply:
```suggestion
```

woops this line is redundant